specification.md

```diff
@@ -1296,12 +1301,9 @@ see the [Workflow Error Handling section](#Workflow-Error-Handling).
 
 ```json
 {
-   "expression": "{{ $.exceptions }}",
-   "errorDataFilter": {
-     "dataOutputPath": "{{ $.exceptions[0] }}"
-   },
+   "error": "Item not in inventory",
```

Comment:
I think we should guide the reader to the use of Swagger. Of course, the error can be domain-specific, i.e. for a different action kind, we'd allow arbitrary strings. But it would be cool if we could use capital camel case error names such as "ItemNotInInventory" to hine to the use of [objects in Swagger responses](https://swagger.io/specification/#components-object)

Reply:
we cant tie error handling to swagger. as mentioned they are not related. the openapi definition will still get executed by a runtime which might use different libs and programming languages to invoke services/operations on the swagger defs. as such the errors those runtimes create might have nothing to do with the actual definition in openapi.

so the mapping has to be be provided by the runtimes in this case.

Reply:
typically on the runtime side this is done by wrapping the underlying exception into a custom one which includes the error code or the defined error name. Then it can be related to the whats specified in the workflow definition